x/posts/internal/types/post.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 // ---------------
-// --- Post id
+// --- Post newID
 // ---------------
 
-// PostID represents a unique post id
+// PostID represents a unique post newID
```

Comment:
I think this is a refactoring error

```suggestion
// PostID represents a unique post id
```